src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingFlorist.java

```diff
@@ -221,7 +222,9 @@ public ItemStack getFlowerToGrow()
             case 4:
             case 5:
             default:
-                return IColonyManager.getInstance().getCompatibilityManager().getCopyOfPlantables();
```

Comment:
that should probably be filtered in the copyOfPlantables creation

Reply:
I thought that was used in the compost section but you are correct